drivers/usb/udc/Kconfig.dwc2

```diff
@@ -45,3 +45,12 @@ config UDC_DWC2_MAX_QMESSAGES
 	default 8
 	help
 	  DWC2 maximum number of ISR event messages.
+
+config UDC_DWC2_USBHS_VBUS_READY_TIMEOUT_MS
+	int "UDC DWC2 USBHS VBUS ready event timeout in ms"
+	depends on UDC_DWC2
```

Comment:
This only applies to USBHS and does not apply to all DWC2 otg controllers.

Reply:
I've described it in this Kconfig help. Do you see any better option? I don't see any symbol that I could depend on.

Reply:
`NRFS_HAS_VBUS_DETECTOR_SERVICE` is probably the best.

Reply:
Added the dependency. Please rereview.